baseplate/sidecars/event_publisher.py

```diff
@@ -128,10 +128,13 @@ def publish(self, payload: SerializedBatch) -> None:
 
                 # we should crash if it's our fault
                 response = getattr(exc, "response", None)
-                if response is not None and response.status_code < 500:
+                if response and response.status_code < 500:
```

Comment:
`Response` has a `__bool__` definition that "Returns True if :attr:`status_code` is less than 400.". That's why we were checking `is not None` instead of truthiness earlier. I think that still needs to be around so this can ever be true for 4xx errors.